src/drive/dal.ts

```diff
@@ -8,3 +12,82 @@ export const getTagsFromFileName = (fileName: string) => {
   });
   return retval;
 };
+
+export const listRecentlyChangedFiles = async (pageToken: nextPageToken) => {
+  const fetchPageResult = await fetchPage(pageToken, {
+    orderBy: "modifiedTime desc",
+    q: "mimeType = 'image/gif' and '0BwoBPbVKwbI9TUdFSG0yRjh5UTQ' in parents",
+    pageSize: 5
+  });
+  if (!fetchPageResult.files) return;
+  fetchPageResult.files = filesToTags(fetchPageResult.files);
+  return fetchPageResult as listRecentlyChangedFilesReturn;
+};
+
+interface listRecentlyChangedFilesReturn {
+  nextPageToken: nextPageToken;
+  files: filesToTagsInterface[];
+}
+interface filesToTagsInterface extends drive_v3.Schema$File {
+  tags?: string[];
+}
+const filesToTags = (files: filesToTagsInterface[]) => {
+  for (let i = 0; i < files.length; i++) {
+    const file = files[i];
+    if (!file.name) continue;
+    file.name = file.name.toLowerCase();
+    file.tags = getTagsFromFileName(file.name);
+  }
+  return files;
+};
+
+export const listAllFiles = async () => {
```

Comment:
This looks unused, we can just remove this function since it's capped at three pages so it's not even looking at all the files.